package/aioracle/aioracle_v2/src/contract.rs

```diff
@@ -852,17 +848,22 @@ pub fn get_participant_fee(
     deps: Deps,
     pubkey: Binary,
     service_addr: &str,
+    mut service: Option<String>
 ) -> Result<Coin, ContractError> {
     let Config { denom, .. } = CONFIG.load(deps.storage)?;
     let executor_addr = pubkey_to_address(&pubkey)?;
+    if service.is_none() {
+        service = Some(SERVICE_NAME_DEFAULT.to_string());

```

Comment:
I dont think we should do this. We can just return Err with a custom error message

Reply:
@ducphamle2  Because I want when the old client requests without this "service name" parameter, but leave the default value as "price" so that it can still run, no need to edit code the old client.
If it doesn't suitable, let me throw Error

Reply:
dont worry, with the new provider_bridge contract, there will always be service names

Reply:
@ducphamle2 , i change code: https://github.com/oraichain/oraiwasm/pull/21/commits/2e0984ce2ceec1710fcd7474b4f28570e98d197a